spec/models/spree/order_decorator_spec.rb

```diff
@@ -90,6 +82,22 @@
     end
   end
 
+
+  describe "#adjust_avalara" do
+    context 'committed order' do
+      let(:adjust_order) {
+        order.avalara_capture_finalize
+        order
+      }
+      it "should response with Hash object" do
+        expect(adjust_order.adjust_avalara).to be_kind_of(Hash)
+      end
+      it 'should have a ResultCode of success' do
+        expect(adjust_order.adjust_avalara[:result_code]).to eq('Success')
+      end
```

Comment:
No reason really to have those as separate tests.

the `it` description is a bit redundant to the expectation. Maybe leave it out if you don't have anything more to describe on the test.